### ISSUE TITLE
404 on flags@2x.png due to phone-country.css

### ISSUE THREAD

**Issue by lenvanessen** [verbb/formie#971]
### Describe the bug

Phone-country.css references a file 
`
   .iti__flag {
      background-image: url("../img/flags@2x.png"); } }
      `
But it doesn't exist, it's probably due to the import done here:
https://github.com/verbb/formie/blob/craft-3/src/web/assets/frontend/src/scss/fields/phone-country.scss#L2

Would be nice if it could be fixed

### Steps to reproduce

1. Create a form with a phone field

### Form settings

x

### Craft CMS version

4

### Plugin version

2.0.0

### Multi-site?

no

### Additional context

x

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by lenvanessen** [comment]
@engram-design it isn't fixed here after updating

**Comment by lenvanessen** [comment]
It's also loaded twice by the way, one time from cdnjs @ cloudflare and one time from a local folder /cpresources/img/flags@2x.png


![Schermafbeelding 2022-07-07 om 08 42 44](https://user-images.githubusercontent.com/23024452/177708821-5dfba534-e04e-4ec1-baea-00d2bbdfc194.png)

**Comment by engram-design** [comment]
Strange, only seeing it come up once for me. I did also test on a retina screen, which loads the `flags@2x.png` file.

![image](https://user-images.githubusercontent.com/1221575/177709218-ae363bc8-b44e-4ad2-b8e2-7149f3e2afbc.png)

Cleared all your caches?

**Comment by lenvanessen** [comment]
Hi @engram-design,

was cloudflare's cache:) Fixed, thanks

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/formie/releases/tag/2.0.0)
